From fa73238dfe5e29d9d77ecb2166fbae46215d59b6 Mon Sep 17 00:00:00 2001
From: vauban353 <vauban353@gmail.com>
Date: Sat, 12 Aug 2023 18:14:01 +0100
Subject: [PATCH] BeagleV-Fire: Add printk to IM219 driver for board tests.

---
 drivers/media/i2c/imx219.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/drivers/media/i2c/imx219.c b/drivers/media/i2c/imx219.c
index 7a14688f8c22..effb399b143f 100644
--- a/drivers/media/i2c/imx219.c
+++ b/drivers/media/i2c/imx219.c
@@ -1181,6 +1181,9 @@ static int imx219_identify_module(struct imx219 *imx219)
 	int ret;
 	u32 val;
 
+	printk(KERN_INFO "imx219_identify_module()\n");
+
+
 	ret = imx219_read_reg(imx219, IMX219_REG_CHIP_ID,
 			      IMX219_REG_VALUE_16BIT, &val);
 	if (ret) {
@@ -1195,6 +1198,9 @@ static int imx219_identify_module(struct imx219 *imx219)
 		return -EIO;
 	}
 
+	printk(KERN_INFO "imx219_identify_module() - Success\n");
+
+
 	return 0;
 }
 
@@ -1402,6 +1408,8 @@ static int imx219_probe(struct i2c_client *client)
 	struct imx219 *imx219;
 	int ret;
 
+	printk(KERN_INFO "imx219_probe()\n");
+
 	imx219 = devm_kzalloc(&client->dev, sizeof(*imx219), GFP_KERNEL);
 	if (!imx219)
 		return -ENOMEM;
-- 
2.25.1

